front/src/views/profile/entertainments/entertainments.view.html

```diff
@@ -43,6 +43,16 @@
           :isActive="selectedPlaceIndex && selectedPlaceIndex === index + 1 ? true : false"
```

Comment:
:isActive="selectedPlaceIndex && selectedPlaceIndex === index + 1"

result is bool, why you need return true or false?